config.js

```diff
@@ -2,11 +2,30 @@
 
 module.exports = {
   env: process.env.NODE_ENV || 'local',
-  bankHolidayApi: 'https://www.gov.uk/bank-holidays.json',
-  maxPayloadSize: process.env.MAX_PAYLOAD_SIZE || '100kb',
-  http_port: process.env.PORT || 3001,
-  https_port: process.env.HTTPS_PORT,
   requestTimeout: +process.env.REQUEST_TIMEOUT || 1000,
-  serviceName: process.env.SERVICE_NAME,
-  latestMigration: process.env.LATEST_MIGRATION
+  db: {
+    host: process.env.DB_HOST || 'localhost',
+    port: process.env.DB_PORT || '5432',
+    user: process.env.DB_USER || 'postgres',
+    password: process.env.DB_PASS || 'postgres',
+    database: process.env.DB_NAME,
+    client: process.env.DB_CLIENT || 'pgp',
+    model: process.env.DB_MODEL || 'pgp-model'
+  },
+  service: {
+    serviceName: process.env.SERVICE_NAME,
+    targetTable: process.env.TARGET_TABLE,
+    sourceFileTable: process.env.SOURCE_FILE_TABLE
+  },
+  keycloak: {
+    tokenUrl: process.env.KEYCLOAK_TOKEN_URL,
+    username: process.env.KEYCLOAK_USERNAME,
+    password: process.env.KEYCLOAK_PASSWORD,
+    clientId: process.env.KEYCLOAK_CLIENT_ID,
+    secret: process.env.KEYCLOAK_SECRET
+  },
+  notify: {
+    notifyKey: process.env.NOTIFY_KEY || 'test-key',
+    caseworkerEmail: process.env.CASEWORKER_EMAIL
+  }
```

Comment:
Do we not want to avoid defaults as we have had issues with defaults in production

Reply:
Defaults removed.